lib/logs.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:logs/src/logging_service.dart';
```

Comment:
Perhaps have this be a minimal file that just exports `lib/src/logs.dart`?

Reply:
Good idea.

Will do.